src/components/ShowMoreScholarships.jsx

```diff
@@ -0,0 +1,24 @@
+import { useContext } from 'react';
+import ScholarshipsContext from '../models/ScholarshipsContext';
+import Grid from '@mui/material/Grid';
+import ScholarshipCard from './ScholarshipCard';
+
+export default function ShowMoreScholarships({ currentId }) {
```

Comment:
(optional) Since this component is small you could inline it in the ViewScholarhip file.